src/ovp_pipeline/commands/digest_handler.py

```
@@ -54,382 +76,557 @@
+        + sources
+        + footer
+    )
 
-    Obsidian-flavoured wikilinks so clicking inside Obsidian jumps
-    straight to the source.  In the ovp-ui ``/note`` renderer the
-    same wikilinks resolve via ``_replace_wikilinks_with_markdown_links``,
-    so the same markdown works in both surfaces.
 
-    Empty when no inputs carry source ids (e.g. on the empty-vault
-    stub digest), so the digest doesn't render an empty heading.
+def _render_no_data_body(inputs: DigestInputs, pack: str) -> str:
```

Ruff (0.15.12)

[warning] Unused function argument: `pack`

(ARG001)